### DISCUSSION TITLE
https://maxkim-j.github.io/posts/what-type-of-developer-am-i

### DISCUSSION THREAD

**Discussion by MaxKim-J** [MaxKim-J/maxkim-j.github.io#82] [Announcements]
# 김맥스 기술 블로그 | 나는 어떤 개발자일까

https://maxkim-j.github.io/posts/what-type-of-developer-am-i

**Comment by MaxKim-J** [comment]
<p>👍👍👍👍👍👍👍👍👍👍👍👍👍</p>

>_This comment was made by Disqus. 2021-08-31 04:12:00 라이언(disqus_Bam85l2bkM)_

**Comment by MaxKim-J** [comment]
<p>일로써 개발 시작한지는 3년차지만, 웹 개발 분야(JS, TS, React)는 올해 처음접해서 좌충우돌 하고있습니다,</p><p>react-query 글 보러 들어왔다가 다른 글들도 보게되었는데,</p><p>제가 추구하는 방향과 비슷하시군요.</p><p>아직 많이 부족하지만 '구도자'의 마음으로 더 정진해야겠습니다.</p>

>_This comment was made by Disqus. 2021-09-14 09:47:22 Young Joo Kim(disqus_ZYfEhcNyX7)_

**Comment by Taeeon-kim** [comment]
좋은글 감사합니다. 1년차 개발자로써 어떤 계기로 개발자가 되었고 무슨 개발자가 되길 원하는지 생각하며 글을 읽게 되었습니다. 개발자만의 철학이 무엇인지는 개인마다 다르다고는 알고 있는데 일맥상통하는 부분이 많은것 같습니다. 앞으로도 자기의 일을 즐길수있는 개발자가 되길 화이팅

**Comment by SnowLeopardMoonBear** [comment]
안녕하세요, 4년차 백엔드 개발자고요, DevOps로의 재취업을 준비하고 있습니다. 입사 지원에 앞서 나는 어떤 개발자일지 고민하다가 우연히 들어왔어요. 우선 공감가는 좋은 글 써주셔서 감사합니다. 서비스의 큰 그림을 그리고, 더 나은 세상을 그린다는 점에서 생각이 비슷해 반가웠습니다. 

무엇보다도 복잡성을 대하는 사유과 태도가 무척이나 인상깊었어요. 이미 아실 지도 모르겠지만 [선과 모터사이클 관리술]이라는 책을 추천드립니다. 현대 기술 문명이 기하급수적으로 복잡해지는 가운데, "과연 무엇에 집중하고 무엇을 추구할 것이냐"는 물음에 답하는 철학 소설이에요.

혹시 괜찮으시다면 편하게 메일 주시면 감사하겠습니다. 개발이든 뭐든 같이 얘기 나눠보면 좋겠다는 생각이 들어서 초면에 염치불구하고 말씀드렸습니다. 그럼 오늘도 좋은 하루 보내세요!
